perses/app/experiments.py

```diff
@@ -52,7 +52,8 @@ class BuildProposalNetwork(object):
                          'hmass': 4 * unit.amus,
                          'map_strength': 'default',
                          'phases': ['vacuum', 'solvent', 'complex'],
-                         'forcefield_files': ['gaff.xml', 'amber14/protein.ff14SB.xml', 'amber14/tip3p.xml'],
+                         'forcefield_files': ['amber/protein.ff14SB.xml', 'amber/tip3p_standard.xml', 'amber/tip3p_HFE_multivalent.xml'],
+                         'small_molecule_forcefield' : 'openff-1.0.0',
```

Comment:
is this `gaff`?

Reply:
No, this is the `openforcefield 1.0.0` ("Parsley") force field. 

If you'd rather the default be `gaff-2.11`, just change this to `gaff-2.11`. No preference here at the moment!